dask/array/tests/test_overlap.py

```diff
@@ -388,7 +388,7 @@ def test_map_overlap_multiarray_defaults():
     z = da.map_overlap(lambda x, y: x + y, x, y)
     # func should be called twice and get (5,) and (1, 5) arrays of ones each time
     assert_eq(z.shape, (1, 10))
-    assert_eq(z.sum(), 20)
```

Comment:
This is a real fix! Since `20` doesn't have a `dtype` we weren't catching that the output of `z.sum()` is a float not an int.